scripts/src/chartprreview/chartprreview.py

```diff
@@ -14,27 +14,34 @@
 except ImportError:
     from yaml import Loader, Dumper
 
+def write_error_log(directory, *msg):
+    with open(os.path.join(directory, "errors"), "w") as fd:
+        for line in msg:
+            print(line)
+            fd.write(line)
+            fd.write("\n")
 
-def get_modified_charts(api_url):
+def get_modified_charts(directory, api_url):
     files_api_url = f'{api_url}/files'
     headers = {'Accept': 'application/vnd.github.v3+json'}
     r = requests.get(files_api_url, headers=headers)
     pattern = re.compile(r"charts/(\w+)/([\w-]+)/([\w-]+)/([\w\.]+)/.*")
-    count = 0
     for f in r.json():
         m = pattern.match(f["filename"])
         if m:
             category, organization, chart, version = m.groups()
             return category, organization, chart, version
 
-    return "", "", "", ""
+    msg = "[ERROR] The files modified are not part of any chart"
```

Comment:
Maybe "One or more files included in the pull request are not part of the chart"

Reply:
Done. 229bd4e5c3b304f1c759c7b5b8f51971defae155